src/compositor/seat/seat.c

```diff
@@ -115,7 +115,7 @@ seat_handle_key(struct wlc_seat *seat, const struct wlc_input_event *ev)
 
    wlc_keyboard_update_modifiers(&seat->keyboard, ev->device);
 
-   if (seat->keyboard.modifiers.mods == (WLC_BIT_MOD_CTRL | WLC_BIT_MOD_ALT) && ev->key.code >= 59 && ev->key.code <= 88) {
+   if (seat->keyboard.modifiers.mods == (WLC_BIT_MOD_CTRL | WLC_BIT_MOD_ALT) && ev->key.code >= 59 && ev->key.code <= 70) {
```

Comment:
The XKB symbols should work, but you can't use the .code as it means raw keycode and this may change depending on keyboard used. E.g. for me F12 == 96 (X11 historically adds 8 to keycodes, so 88 here). So this would break the current F range for me.